### PR TITLE
Create gg.js

### PR DESCRIPTION
**Pull request by 0xSullivan**

### CODE REVIEW

**Comment by bakhshandeh** [comment]
Let’s use a better name.
Project-name_app-name

File: general/gg.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+const { BigNumber } = require("ethers")
```
Comment (bakhshandeh) [inline general/gg.js:1]: Use BN from MuonAppUtils

File: general/gg.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+					currentTotal += await ethCall(
+						LGE_ADDRESSES[chainId],
+						'totalSoldToken',
+						[],
+						LGE_ABI,
+						chainId
+					)
+				}
+
+				if (timestamp > getTimestamp()) throw { message: 'Invalid Timestamp' }
```
Comment (bakhshandeh) [inline general/gg.js:91]: Timestamp could be too old.
Is it fine ?

Comment (0xSullivan) [reply general/gg.js:91]: I will change it, thank you

**Comment by 0xSullivan** [comment]
Changes applied

**Pull request merged**